src/common/prod/locals.tf

```diff
@@ -36,6 +36,10 @@ locals {
     }
   }
 
+  cosmos_api = {
+    allowed_subnets = ["fn3admin", "fn3app1", "fn3app2", "fn3appasync", "fn3assets", "fn3public", "fn3services", "fn3slackbot"]
```

Comment:
Mmmh we're creating a dependency to other configs..
However, there is no concrete solution.
Moreover, this Cosmos has the private endpoint enabled, so these firewall exception should be useless. We can face this issue during the migration of these functions